test/e2e/installplan_e2e_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
I think we're missing imports for the following that weren't part of this backport patch:
```Go
k8sjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
"k8s.io/apimachinery/pkg/runtime"
"bytes"
```

Reply:
Thanks, opened #1963 instead